packages/nimble-components/src/table/tests/table.stories.ts

```diff
@@ -17,18 +17,26 @@ interface TableArgs {
 
 const simpleData = [
     {
-        myStr: 'my first row',
-        myNum: 5,
-        myBool: true
+        firstName: 'Ralph',
+        lastName: 'Wiggum',
+        favoriteColor: 'Rainbow',
+        quote: "I'm in danger!"
     },
     {
-        myStr: 'my second row',
-        myNum: 15,
-        myBool: false
+        firstName: 'Milhouse',
+        lastName: 'Van Houten',
+        favoriteColor: 'Crimson',
+        quote: "Not only am I not learning, I'm forgetting stuff I used to know!"
+    },
+    {
+        firstName: 'Ned',
+        lastName: 'Flanders',
+        favoriteColor: 'Taupe',
+        quote: 'Hi diddly-do neighbor!'
```

Comment:
https://www.youtube.com/watch?v=o_YBngq5mc0&t=1s
```suggestion
        quote: 'Hi diddly-ho neighbor!'
```